settings/DevHome.Settings/Views/AccountsPage.xaml.cs

```diff
@@ -58,7 +60,24 @@ private async void AddAccountButton_Click(object sender, RoutedEventArgs e)
         var numProviders = ViewModel.AccountsProviders.Count;
         if (numProviders == 1)

```

Comment:
Why the seperation between one provider and more than one provider?  What is stopping from putting this in a loop?

Reply:
This change was added to streamline the add button to no longer show a flyout if one provider is present (more context here: https://github.com/microsoft/devhome/pull/339). Thanks!